README.md

```diff
@@ -1,12 +1,82 @@
-# Bouncer
+<div id="top"></div>
+<!--
+*** Thanks for checking out the Best-README-Template. If you have a suggestion
+*** that would make this better, please fork the repo and create a pull request
+*** or simply open an issue with the tag "enhancement".
```

Comment:
GitHub account without Contributor access to the repository, the creator of the issue cannot set tags on this issue.
Only selecting a template can allow the user to set tags.

Reply:
Thats the comment of the Template i used. I can remove it if it's needed

Reply:
I think you can just remove the following to be more clear for future contributor :
```suggestion
*** or simply open an issue.
```